src/components/Images.vue

```diff
@@ -274,54 +277,176 @@ export default {
       this.startRunningCommand('push');
       ipcRenderer.send('do-image-push', obj.imageName.trim(), obj.imageID.trim(), obj.tag.trim());
     },
     doBuildAnImage() {
+      if (this.highlightExistingImage(this.imageToBuild)) {
+        return;
+      }
       this.currentCommand = `build ${ this.imageToBuild }`;
       this.fieldToClear = 'imageToBuild';
+      this.postCloseOutputWindowHandler = this.scrollToNewImage(this.imageToBuild);
       this.startRunningCommand('build');
       ipcRenderer.send('do-image-build', this.imageToBuild.trim());
     },
     doPullAnImage() {
+      if (this.highlightExistingImage(this.imageToPull)) {
+        return;
+      }
       this.currentCommand = `pull ${ this.imageToPull }`;
       this.fieldToClear = 'imageToPull';
+      this.postCloseOutputWindowHandler = this.scrollToNewImage(this.imageToPull);
       this.startRunningCommand('pull');
       ipcRenderer.send('do-image-pull', this.imageToPull.trim());
     },
+    /**
+     * If the named image is loaded, scroll to it and return true.
+     * Otherwise return false.
+     * @param fullImageName {string{}}
+     * @returns {boolean}
+     */
+    highlightExistingImage(fullImageName) {
+      const [imageName, tag] = this.parseFullImageName(fullImageName);
+
+      const image = this.getImageByNameAndTag(imageName, tag);
+
+      if (image) {
+        const options = {
+          message:   `Image ${ fullImageName } is already loaded`,
+          type:      'warning',
+          title:     `Ignoring request`,
+        };
+
+        ipcRenderer.invoke('show-message-box-sync', options).then((_) => {
+          this.scrollToImage(image);
+        });
+
+        return true;
+      }
+
+      return false;
+    },
+    /**
+     * syntax of a fully qualified tag could start with <hostname>:<port>/
+     * so a colon precedes a tag only if its followed only by valid tag characters
+     * @param fullImageName {string}
+     * @returns {[string, string]}
+     */
+    parseFullImageName(fullImageName) {
+      const m = /^(.+?):([-._A-Za-z0-9]+)$/.exec(fullImageName);
+
+      return m ? [m[1], m[2]] : [fullImageName, 'latest'];
+    },
+    getImageByNameAndTag(imageName, tag) {
+      return this.images.find(image => image.imageName === imageName && image.tag === tag);
+    },
+    scrollToImage(image) {
+      const row = this.$refs.imagesTable.$el.querySelector(`tr[data-node-id="${ image.imageID }"]`);
+
+      if (row) {
+        this.$nextTick(() => {
+          row.scrollIntoView();
+          row.classList.add('greenfade');
+        });
+        setTimeout(() => {
+          row.classList.remove('greenfade');
+        }, 3_000);
+      } else {
+        console.log(`Can't find row for ${ image.imageName }:${ image.tag } in the image table`);
+      }
+    },
+    scrollToNewImage(imageToPull) {
+      return () => {
+        if (this.imageManagerOutput.trimStart().startsWith('Error:')) {
```

Comment:
This makes me thing `scrollToNewImage` isn't quite the right name for this function (or it needs a doc comment describing this behaviour).

Reply:
Changed to `findAndScrollToImage`